### DISCUSSION TITLE
Some English words are detected as German/French

### DISCUSSION THREAD

**Discussion by thirtha-prasad** [pemistahl/lingua-py#131] [General]
[en_results.csv](https://github.com/pemistahl/lingua-py/files/10892236/en_results.csv)

Attached are the results of some English strings/words captured from my web page. While detecting the language using lingua few words are showing as German or French. 

I need to validate the same for French and Germen as well. I mean the case is, I will capture all the strings from web page and verify in which language they are in. Expected is they should be in the language set in the application or browser language.

When looking at the above result, it is been confused that for some words it is detecting wrongly. Can you please check if any other way that i can get the more nearest results.

**Comment by pemistahl** [comment]
Pure statistical approaches to language detection are never 100% correct. This is not a bug within the library. Feed longer strings into the detector and you will get more reliable results.

You are obviously able now to use the library. Did you solve your installation problems?

**Reply by thirtha-prasad** [reply]
I am able to use the library in my windows machine perfectly, But same thing if i run through CircleCI in a docker node its not working.

**Reply by pemistahl** [reply]
Then it's probably a docker issue. I cannot help you with that, sorry.

**Comment by thirtha-prasad** [comment]
Instead of Python, Do you have the same library in JS or Cypress?

**Reply by pemistahl** [reply]
No, I have implemented Lingua in Python, Kotlin, Go and Rust. As soon as the next release of the Rust one is finished, there will be a JavaScript version available by compiling it to WebAssembly (WASM).

**Reply by thirtha-prasad** [reply]
Do you have any schedule or a tentative timelines for JS

**Reply by pemistahl** [reply]
No, I haven't. It's done when it's done.
